spacewalk/certs-tools/spacewalk-certs-tools.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Sun May 2 14:08:00 GMT 2021 digdilem@gmail.com
+- Prepare the bootstrap script generator for Rockylinux 8
+
+
+-------------------------------------------------------------------
```

Comment:
For changelogs please add just the line with the change on top of the file.
The "Header" will be added by our tooling when we submit the change to the build service.
```suggestion
- Prepare the bootstrap script generator for Rockylinux 8
```